5calls/app/src/main/res/xml/settings.xml

```diff
@@ -48,6 +48,12 @@
         android:title="@string/settings_category_general"
         >
 
+        <EditTextPreference
+            android:key="prefsKeyUserName"
+            android:singleLine="true"
+            android:summary="@string/settings_name_description"
+            android:title="@string/settings_name" />
```

Comment:
Please put the /> on a new line -- it reduces diffs if we add attributes to this XML later.